From be1e1b33191afdcfe3c2ecc4ff3e361a5859e9c6 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Fri, 30 Jan 2015 00:40:22 -0800
Subject: [PATCH] contrib.mak: fix for parallel build

Fixed:
rm: cannot remove `/usr/share/ghostscript/9.15/lib': Is a directory

Create lib before install to fix the race issue.

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 contrib/contrib.mak |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/contrib/contrib.mak b/contrib/contrib.mak
index 08a80d1..de2e20d 100644
--- a/contrib/contrib.mak
+++ b/contrib/contrib.mak
@@ -947,6 +947,7 @@ $(DEVOBJ)dviprlib.$(OBJ) : $(JAPSRC)dviprlib.c $(JAPSRC)dviprlib.h
 	$(DEVCC) $(O_)$@ $(C_) $(JAPSRC)dviprlib.c
 
 extra-dmprt-install:
+	mkdir -p $(DESTDIR)$(gsdatadir)$(D)lib
 	$(INSTALL_DATA) $(JAPSRC)dmp_init.ps $(DESTDIR)$(gsdatadir)$(D)lib || exit 1
 	$(INSTALL_DATA) $(JAPSRC)dmp_site.ps $(DESTDIR)$(gsdatadir)$(D)lib || exit 1
 	$(INSTALL_DATA) $(JAPSRC)escp_24.src $(DESTDIR)$(gsdatadir)$(D)lib || exit 1
@@ -1088,6 +1089,7 @@ $(DEVOBJ)gdevalps.$(OBJ) : $(JAPSRC)gdevalps.c $(PDEVH)
 ### ----------------- Additional .upp files ---------------- ###
 
 extra-upp-install:
+	mkdir -p $(DESTDIR)$(gsdatadir)$(D)lib
 	for f in $(CONTRIBSRC)uniprint$(D)*.upp; do \
 	    $(INSTALL_DATA) $$f $(DESTDIR)$(gsdatadir)$(D)lib || exit 1; \
 	done
-- 
1.7.9.5

